bindings/node/index.d.ts

```diff
@@ -404,46 +404,45 @@ export class ClientEncryption {
     callback: ClientEncryptionCreateDataKeyCallback
   ): void;
 
-  /** @experimental */
+  /** 
+   * Searches the keyvault for any data keys matching the provided filter.  If there are matches, the attempts to re-wrap the data keys using the provided options.
```

Comment:
```suggestion
   * Searches the keyvault for any data keys matching the provided filter.  If there are matches, attempts to re-wrap the data keys using the provided options.
```

Reply:
I didn't phrase it exactly like you suggested, but I did fix the typo.  That comment was also present in the `js` source file, so I fixed it there and regenerated the readme.